### DISCUSSION TITLE
임포트문을 적을 때 각 섹션(표준 라이브러리, 서드 파티, 사용자 정의)을 알파벳 순서로 적는 것이 좋을까요?

### DISCUSSION THREAD

**Discussion by yunjjun** [8percent/styleguide#43] [Q&A] [answered]
PEP 8 에서 제시하고 있는 것 같은데, 저희가 초안을 작성할 때 추가하지 않은 이유나 이전의 논의가 있을까요?
그리고 임포트문 말고도 일반적으로 알파벳 순서를 따르는 것을 권장하는 경우가 있는지 궁금합니다.

**Comment by cbscsm** [comment, marked as answer]
저희 isort에 의해서 알파벳 순으로 정리되는 것으로 알고 있는데, 혹시 예외가 있나요?
